### DISCUSSION TITLE
Open up for group discussion: Drug Therapy

### DISCUSSION THREAD

**Discussion by rtmill** [OHDSI/OncologyWG#491] [Ideas]
# Problem Space

Model representation and vocabulary for oncology related drug therapies, including:
-Chemotherapy
-Immunotherapy
-Hormone Therapy 

# Factors
- Regimen
- Vocabularies
- Relation to drug exposures
- Dosage
- Phase/Course
- Intent
- Others?

# Proposed Solutions

This one is a bit trickier than others as we need to account for a wide variety of representations in the source data without losing context. Specifically, when the source data contains the linkages between the regimen and the events vs. when the source data only contains the events. 

Through our discussions, the baseline of what could work for both circumstances is:
- Use HemOnc concepts to represent regimen
- Keep individual drug exposure records as drug exposure records
- Change domain of Regimen to Procedure
- Use modifiers to fill in details of the procedure (see factors above) 

This is also a bit of a loaded topic given it crosses into the Episode domain. There have been extensive discussions around that, specifically if episodes should be derived only or if they can be created during the ETL. The most recent (and lengthy) discussion around that can be found here: https://forums.ohdsi.org/t/2022-us-symposium-66-episode-and-episode-event-tables-documentation/17501

If we were to enable both paths, while also maintaining that details are populated at the lower level, that could look like:

1) There is sufficient relational data in your source to justify creating Treatment episodes during ETL. In this circumstance:
- the ETL developer determines the creation of the episode
- treatment episode ties to a Procedure for the regimen
- episode_event table links the drug (and procedure if combination therapy) records to the treatment episode

2) The source data either does not contain that level of connectivity or detail, but instead contains drug therapy data represented as only drug exposures. 
- drug exposures stay as drug exposures
- regimen derivation software is used to extract regimen from the drug exposure records
- if regimen do exist in the source data, but without the connection to the individual drug records, they are procedures without linkages

The issue arises that if we want to represent the connection between regimen and drug exposures at the lower level (outside of episodes) and those relations have been determined, how can we (and should we at all) maintain the link between the two? In other words, if you run a regimen detector algorithm on your drug exposures, and it creates those new procedure records with a type along the lines of 'derived', where is the linkage between the drug record and procedure record stored? Fact relationship?

**Comment by askanter** [comment]
I am interested in this discussion because it has relevance for HIV treatments as well. Many systems track progress on regimens which are made up of individual drug prescriptions. However, the clinical data is often tied to the regimen mnemonic not the individual drugs. Pharmacy data is likely to be individual, but the clinicians don't document that way. To top it off, there are classes of regimens (first line, second line, etc.) which need to be tracked. So changing from one line to another is important, not just that the regimen changed due to intolerance in one of the components. Having a regimen concept allows for the attachment of metadata to the group of drugs (such as line, or start/stop dates, reasons of change, etc.). Managing the maps between Regimens and their component drugs (with their component RxNORM codes) is a challenge since the regimen might be made of different combinations of drugs (A+B+C, A/B+C, A+B/C or even a single drug A/B/C).

**Comment by gkennos** [comment]
We have done a bit of work in this space as we were doing some concerted effort in mapping eviQ and HemOnc protocols, so needed to have some data ready to test this in.

They way in which we handled this was to store the act of prescribing a care-plan as a procedure in and of itself, irrespective of any actual drug delivery. These procedure_occurrence records have a procedure_concept_id of one of our custom eviQ concept IDs* (most of which have a mapping relationship to a HemOnc concept) and a procedure_type_concept_id of ‘EHR planned dispensing record’ (32837).

We are in the fortunate position that the eventual drug delivery events are linked back to this prescription event in our source system, so when building the episode, we can assign both this care-plan event and the drug exposure records back to the treatment episode as episode events. This becomes extremely useful when seeing if patients completed their course of treatment or not, or even if they had a dose modification, as we are able to link the drug exposure -> treatment episode -> disease episode, and we link the best-available dose-basis-relevant observations to the disease episode so can calculate ‘treatment as intended’ versus ‘treatment as delivered’ from that**.


\* these are currently custom but have confirmed with the CINSW that licensing of eviQ is creative commons so they are generally-speaking supportive of creating it as a vocab if there is interest, and certainly in the meanwhile, happy to share with anyone who wants to use

** this calculation also requires the use of some custom tables to hold the full drug-delivery prescription including dosage and schedule as specified for each eviQ regimen, which sits outside of the current model. There are some more complex regimens with branching logic and alternative drugs etc. where it may fall over but performs well for the bulk of them.

**Reply by gkennos** [reply]
I do also definitely agree that this has weight on whether or not episodes are derived only or created during ETL. We found that the care plan assignment to drug exposure link had nowhere in the data model to live other than the episode level, which meant that we had to pull episode creation forward in the process to accommodate, rather than assembling post-hoc. (I had initially been in camp-derive-only prior to this issue, and couldn't find any suitable workaround at all)
